src/SFML/Window/Unix/ClipboardImpl.cpp

```diff
@@ -143,12 +141,11 @@ String ClipboardImpl::getStringImpl()
     const Clock clock;
 
     // Wait for a response for up to 1 second
-    while (!m_requestResponded && (clock.getElapsedTime() < sf::seconds(1)))
```

Comment:
Is `m_requestResponded` still needed if we can just remove it here?

Feels more like there's either a missing update or a threaded update that's not considered.

Reply:
I'm not sure why it was there in the first place, but `m_requestResponded = false;` is right above, making any check meaningless.

Reply:
Fine, I'll check then 😄

Since we're not reading the `m_requestResponded` anywhere else in ClipboardImpl.cpp, the member variable can be fully removed.
It's been like this since the refactoring by @binary1248 in #1437. Seems like some left over from a refactoring step, before going full on multi-threading magic.

Reply:
I recreated the `sf::Clipboard` test failures locally. Reverting these changes fixed those failing tests.